lib/supabase/getArtistKnowledge.ts

```diff
@@ -0,0 +1,30 @@
+import supabase from "./serverClient";
+
+export interface KnowledgeBaseEntry {
+  url: string;
+  name: string;
+  type: string;
+  content?: string;
+}
+
+export async function getArtistIdForRoom(roomId: string): Promise<string | null> {
+  const { data, error } = await supabase
+    .from("rooms")
+    .select("artist_id")
+    .eq("id", roomId)
+    .single();
+  
+  if (error || !data?.artist_id) return null;
+  return data.artist_id;
+}
```

Comment:
Single Responsibility Principle
- actual: 2 functions defined in the same file
- required: 2 files, one for each function